sdk/nodejs/src/cloudClient.ts

```diff
@@ -22,6 +22,15 @@ const castIncomingUser = (user: DVCUser) => {
     return user
 }
 
+const throwIfUserError = (err: AxiosError, isVariableRequst?: boolean) => {
+    const data = err.response?.data as { message: string, statusCode: number }
+    const code = err.response?.status || 0
+
+    if (code !== 404 && code < 500) {
+        throw new Error(`Bucketing request failed with status ${code}. ${data.message || ''}`)
```

Comment:
this error message isn't accurate in the case where you called Track or something, which isn't really a "bucketing" request. Maybe this could just say `DevCycle request failed` or something, unless you want to include the name of the actual method that was being called, like `allVariables Request failed`

Reply:
good point, i'll use `DevCycle request failed` since the method should be in the stack trace